### ISSUE TITLE
#define only likes 1 whitespace

### ISSUE THREAD

**Issue by Wyqer** [SkaceKamen/vscode-sqflint#40]
I don't know how long this small issue persists, I just recognized it some days ago.
The preprocessor command #define is causing the linter to throw an error, when there is more than one whitespace after the command. Guess it's a small and easy fix. 🙂 

```sqf
// This shows errors in the linter
#define        bla        1

// this is fine
#define bla            1

// it doesn't accept more then one whitespace after #define
```

![fine](https://user-images.githubusercontent.com/3811977/47704777-0f190a00-dc25-11e8-81d0-c9479a63bd4d.png)

![notfine](https://user-images.githubusercontent.com/3811977/47704783-12ac9100-dc25-11e8-9f4a-5a2055e71bcc.png)

![notfinetooltip](https://user-images.githubusercontent.com/3811977/47704788-15a78180-dc25-11e8-9810-0813c1ef7366.png)

**Comment by SkaceKamen** [comment]
Thanks for reporting this, the code right now doesn't count on more than one space.

The sqf preprocessor is bit of a mess right now, fixing it is my next priority.

**Comment by SkaceKamen** [comment]
Fixed in 0.9.5
